src/ui/steps/ClearAllLabelsStep.h

```diff
@@ -0,0 +1,21 @@
+//
+// Created by Anton O. on 1/17/22.
+//
+
+#ifndef TASKMANAGER_SRC_UI_STEPS_CLEARALLLABELSSTEP_H_
+#define TASKMANAGER_SRC_UI_STEPS_CLEARALLLABELSSTEP_H_
+
+
+#include "Step.h"
+#include "utilities/StepUtils.h"
+#include "ui/actions/ClearAllLabelsOfTaskAction.h"
+
+class ClearAllLabelsStep : public Step {
+public:
+    using Step::Step;
```

Comment:
Is there a state in the interface `Step`?

Reply:
yes, will add a layer and fix in another PR